src/content/index.mdx

```diff
@@ -9,18 +9,22 @@ index: 0
 
 Neural Magic's Deep Sparse Platform provides models and tools needed to create sparse models and a CPU-based inference engine that runs sparse models at GPU speeds.
```

Comment:
```suggestion
The Neural Magic Platform provides models and tools needed to create sparse models and a CPU-based inference engine that runs sparse models at GPU speeds.
```

Reply:
We are changing the name from Deep Sparse Platform to Neural Magic Platform FYI.